v2/getting-started/composer.md

```diff
@@ -1,6 +1,51 @@
 # Composer: the what, why, and how
```

Comment:
We need to give this page a shorter title, cause it doesn't fit into the sidebar.